### PR TITLE
[Case 6491] Part 1 of entitySelectionTool clean up

### PR DESCRIPTION
**Pull request by 1P-Cusack**
This is the initial pull request to address [FogBugz Case 6491](https://highfidelity.fogbugz.com/f/cases/6491/edit-js-work-proposed-by-1st-playable).  The overall pull request is being broken up into sections to allow for smaller code review chunks; however, if something else is preferred please let me know.  

The current work in addition to this initial phase may be viewed through several preview pull requests located:
[Case 6491: Preview PR 1](https://github.com/1P-Cusack/hifi/pull/2)
[Case 6491: Preview PR 2](https://github.com/1P-Cusack/hifi/pull/3)
[Case 6491: Preview PR 3](https://github.com/1P-Cusack/hifi/pull/4)
[Case 6491: Preview PR 4](https://github.com/1P-Cusack/hifi/pull/5)
[Case 6491: Preview PR 5](https://github.com/1P-Cusack/hifi/pull/6)
[Case 6491: Preview PR 6](https://github.com/1P-Cusack/hifi/pull/8)

Within this initial pull request, user-facing functionality should be unchanged.  Let me know if you see any differences in behavior regarding these places:
* Rotation
* Scaling
* Translating selections
* Multiple Selections

**Note:**  This pull request branch is based on tags/RELEASE-6985, for comparison purposes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2421/HighFidelity-Beta-PR11165-32f2ecb3da7afd26476a7c016c196f1486d9f1cc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2421/HighFidelity-Beta-PR11165-32f2ecb3da7afd26476a7c016c196f1486d9f1cc.exe

**Comment by Nex-Pro** [comment]
testing

**Comment by Nex-Pro** [comment]
### PASS 

All looks good. In desktop and HMD. No surprises.

tested rotation, scaling translating of individual selections and multiple selections.
(multiple selections in desktop only).

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -30,11 +30,15 @@ function objectTranslationPlanePoint(position, dimensions) {
 SelectionManager = (function() {
     var that = {};
 
+    //---------------------------------------
```
Comment (druiz17) [inline scripts/system/libraries/entitySelectionTool.js]: comments like this violates our coding standards.  Check out our coding standards here - https://github.com/highfidelity/hifi/wiki/Coding-Standard. Please change all other comments like this

File: scripts/system/libraries/entitySelectionTool.js [hunk 2]
```diff
@@ -1545,9 +1555,10 @@ SelectionDisplay = (function() {
             translateHandlesVisible = false;
         }
 
-        var rotation = selectionManager.worldRotation;
-        var dimensions = selectionManager.worldDimensions;
-        var position = selectionManager.worldPosition;
+        //NOTE:  These aren't used below here _and_ they're redefs of vars from above
+        //var rotation = selectionManager.worldRotation;
+        //var dimensions = selectionManager.worldDimensions;
+        //var position = selectionManager.worldPosition;
```
Comment (druiz17) [inline scripts/system/libraries/entitySelectionTool.js]: If these are redefs please remove them.

File: scripts/system/libraries/entitySelectionTool.js [hunk 3]
```diff
@@ -3946,11 +3999,13 @@ SelectionDisplay = (function() {
             var result = Overlays.findRayIntersection(pickRay, true, [rotateOverlayTarget]);
 
             if (result.intersects) {
-                var properties = Entities.getEntityProperties(selectionManager.selections[0]);
+                //var selZeroProps = Entities.getEntityProperties(selectionManager.selections[0]);
```
Comment (druiz17) [inline scripts/system/libraries/entitySelectionTool.js]: If this commented out code isn't need, please remove it.

File: scripts/system/libraries/entitySelectionTool.js [hunk 4]
```diff
@@ -3965,7 +4020,7 @@ SelectionDisplay = (function() {
                 });
                 for (var i = 0; i < SelectionManager.selections.length; i++) {
                     var entityID = SelectionManager.selections[i];
-                    var properties = Entities.getEntityProperties(entityID);
+                    //var properties = Entities.getEntityProperties(entityID);
```
Comment (druiz17) [inline scripts/system/libraries/entitySelectionTool.js]: If commented out code isn't need, please remove it.

File: scripts/system/libraries/entitySelectionTool.js [hunk 5]
```diff
@@ -3789,11 +3838,13 @@ SelectionDisplay = (function() {
             var result = Overlays.findRayIntersection(pickRay, true, [rotateOverlayTarget]);
 
             if (result.intersects) {
-                var properties = Entities.getEntityProperties(selectionManager.selections[0]);
+                //var selZeroProps = Entities.getEntityProperties(selectionManager.selections[0]);
```
Comment (druiz17) [inline scripts/system/libraries/entitySelectionTool.js]: If this commented out code isn't need, please remove it.

**Comment by 1P-Cusack** [comment]
Updated to remove commented out code and separator comments as requested.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2546/HighFidelity-Beta-PR11165-cc2ff565ac549de78a7fe43b5ac07b46e267b023.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2546/HighFidelity-Beta-PR11165-cc2ff565ac549de78a7fe43b5ac07b46e267b023.exe

**Pull request merged**